docs/core/project-sdk/msbuild-props.md

```diff
@@ -1582,6 +1568,35 @@ When you use the [MSTest project SDK](../testing/unit-testing-mstest-sdk.md), th
 
 For more information, see [Microsoft.Testing.Platform profile](../testing/unit-testing-mstest-sdk.md#microsofttestingplatform-profile).
 
+## VSTest&ndash;related properties
+
+The following MSBuild properties are documented in this section:
+
+- [IsTestProject](#istestproject)
+- [TestingPlatformDotnetTestSupport](#testingplatformdotnettestsupport)
+- [TestingPlatformShowTestsFailure](#testingplatformshowtestsfailure)
+- [UseVSTest](#usevstest)
+- [MSTestAnalysisMode](#mstestanalysismode)
```

Comment:
This is specific to MSTest analyzers generally, nethier MTP-specific nor VSTest-specific.